contrib/migration/README.md

```diff
@@ -0,0 +1,52 @@
+# BM HCP Migration Script (Alpha)
+
+## Disclaimer
+**This is not supported** as it is now. Eventually this feature will be released using OADP and a custom plugin to cover this functionality.
```

Comment:
Can we rephrase to something like: 

This script is a targeted solution meant as a stop-gap and designed to enable functionality for specific cases as an interim measure. 

Full support will ultimately be provided through OADP as part of the long-term solution.